index.html

```diff
@@ -7,5 +7,8 @@
 </head>
 <body>
   <script src="script.js"></script>
+  <button id="btn-rock">Rock</button>
+  <button id="btn-paper">Paper</button>
+  <button id="btn-scissors">Scissors</button>
```

Comment:
one alternative to what you've defined at the bottom of script.js:
```js
const rockbtn = document.querySelector("#btn-rock");
rockbtn.addEventListener("click", () => {
  console.log("Rock btn");
  // ...
})
```

is to define a function in JS:
```js
function onRockClick(){
  console.log("Rock btn");
  // ...
}
```
And to bind the call with the button in html like so:

```html
<button onclick="onRockClick()">Rock</button>
```

This reduces the amount of JS code you need and might be clearer when you read the HTML file.